slider/beatmap.py

```diff
@@ -2571,6 +2575,15 @@ def parse(cls, data):
             default=1.0,  # taken from wiki
         )
 
+        background = None
+        video = None
+        if 'Events' in groups:
+            for line in groups['Events']:
+                if line.startswith('0'):
+                    background = line.split('\"')[1]
+                elif line.startswith('Video') or line.startswith('1'):
+                    video = line.split('\"')[1]
+
```

Comment:
can there ever be more than one background or video? I supported multiple in https://github.com/llllllllll/slider/pull/69. I only ask because it matters whether we expose `.background` or `.backgrounds` to the user.

Reply:
I did some tests in osu! stable and it seems to only load the first background in the file and for videos im not sure, because it seems to support multiple videos but the second video wouldn't play for me, at most showing a black screen or a single frame.